Rest/Rest/Rest.csproj

```diff
@@ -1,119 +1,21 @@
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>..\RestTests\FitNesseTestRules.ruleset</CodeAnalysisRuleSet>
-    <DocumentationFile>bin\Debug\Rest.xml</DocumentationFile>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>3</WarningLevel>
-    <CodeAnalysisRuleSet>..\RestTests\FitNesseTestRules.ruleset</CodeAnalysisRuleSet>
-    <DocumentationFile>bin\Release\Rest.xml</DocumentationFile>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.12.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
-    <Reference Include="System.Core" />
-    <Reference Include="System.Net.Http" />
-    <Reference Include="System.Web" />
-    <Reference Include="System.Xml.Linq" />
-    <Reference Include="System.Data.DataSetExtensions" />
-    <Reference Include="Microsoft.CSharp" />
-    <Reference Include="System.Data" />
-    <Reference Include="System.Xml" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="ApplicationInfo.cs" />
-    <Compile Include="ContentHandler.cs" />
-    <Compile Include="ContentObjects\ContentObjectFactory.cs" />
-    <Compile Include="ContentObjects\ContentObject.cs" />
-    <Compile Include="ContentObjects\JsonObject.cs" />
-    <Compile Include="ContentObjects\TextObject.cs" />
-    <Compile Include="ContentObjects\XmlObject.cs" />
-    <Compile Include="Model\IRestRequestFactory.cs" />
-    <Compile Include="Model\RestRequest.cs" />
-    <Compile Include="Model\RestRequestFactory.cs" />
-    <Compile Include="RestTester.cs" />
-    <Compile Include="PropertiesForObject.cs" />
-    <Compile Include="Utilities\ExtensionFunctions.cs" />
-    <Compile Include="Model\Injector.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="RestConfig.cs" />
-    <Compile Include="Model\RestSession.cs" />
-    <Compile Include="Model\SessionContext.cs" />
-    <Compile Include="Utilities\FitNesseFormatter.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="FitNesseRules.ruleset" />
-    <None Include="packages.config" />
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
+	<PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Add Json to the framework independent item group (same for both)

Reply:
fixed.